tests/integration/cli/actions/test_select_client_account_for_staking.py

```diff
@@ -20,8 +20,10 @@
 from nucypher.cli.literature import PREALLOCATION_STAKE_ADVISORY
 from nucypher.config.constants import TEMPORARY_DOMAIN
 from tests.constants import YES
+import pytest
 
 
+@pytest.mark.usefixtures('mock_contract_agency')
```

Comment:
A reminder on my comment yesterday on Discord about `usefixtures`:
> Well, sorry for nitpicking, but I also don't have usefixtures in very high estime, for two reasons: 1. Since the fixture name is in a string, it makes refactoring more difficult, and 2. It doesn't work on top of  another fixture
> As a bonus, for some time we had a typo in some of them and we had usefixture instead and pytest didn't complain. This is of course not a problem of usefixtures but it does activate my spidersense
Basically, it makes me think there's no benefit of using it instead of simply passing the fixture name as an argument